solutions/usaco-1061.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: usaco-1061
+source: USACO Bronze 2020 December
+title: Stuck in a Rut
+author: Ryan Chou
+---
+
+[Official Analysis (C++ and Java)](http://www.usaco.org/current/data/sol_prob3_bronze_dec20.html)
+
+## Explanation
+
+Since $x, y \leq 10^9$, we can't simulate each time period. Instead, we should look for signs which allow two cows to intersect. 
+
+One important observation is that cows can't move backwards, so the only way for two cows to collide is if $n[x] > e[x] $ and $n[y] < e[y]$. 
+
+This ensures that the cows are positioned like this:
+
+![](solutions/USACO-dec20-stuckinarut-bronze/intersection_example.png)
```

Comment:
```suggestion
![](USACO-dec20-stuckinarut-bronze/intersection_example.png)
```
i think this path will work? haven't checked though

Reply:
both of them work, but adding `solutions/` makes it show up in the github file preview as well.

Reply:
does it? i see this
![image](https://user-images.githubusercontent.com/63619830/126858340-e98c4792-c07b-4753-b9f4-50ad9a0571a6.png)

Reply:
👍 i changed the file path and it shows up on github preview. (i have no idea where how i saw the image with the solutions/)